rma/wizard/stock_picking_return.py

```diff
@@ -15,8 +15,6 @@ class ReturnPicking(models.TransientModel):
             ("outgoing", "Customers"),
             ("internal", "Internal"),
         ],
-        related="picking_id.picking_type_id.code",
-        store=True,
```

Comment:
If the field is not related anymore, I believe it becomes useless.
Why this change ?

Reply:
In the Odoo logs, I had this warning:
WARNING test14 odoo.fields: stock.return.picking.picking_type_code: selection attribute will be ignored as the field is related

Reply:
@chafique-delli I think the selection attribute is not needed anymore for related field indeed
So you can just leave the related/store attributes and remove the selection attribute, it should work fine